doc/api/single-executable-applications.md

```diff
@@ -27,32 +27,32 @@ tool, [postject][]:
 
 1. Create a JavaScript file:
    ```console
-   $ echo 'console.log(`Hello, ${process.argv[2]}!`);' > hello.js
+   echo 'console.log(`Hello, ${process.argv[2]}!`);' > hello.js
    ```
 
 2. Create a configuration file building a blob that can be injected into the
    single executable application (see
    [Generating single executable preparation blobs][] for details):
    ```console
-   $ echo '{ "main": "hello.js", "output": "sea-prep.blob" }' > sea-config.json
+   echo '{ "main": "hello.js", "output": "sea-prep.blob" }' > sea-config.json
    ```
 
 3. Generate the blob to be injected:
    ```console
-   $ node --experimental-sea-config sea-config.json
+   node --experimental-sea-config sea-config.json
    ```
 
 4. Create a copy of the `node` executable and name it according to your needs:
    ```console
-   $ cp $(command -v node) hello
+   cp $(command -v node) hello
    ```
 
 5. Remove the signature of the binary:
 
    * On macOS:
 
    ```console
-   $ codesign --remove-signature hello
+   codesign --remove-signature hello
    ```
 
    * On Windows (optional):
```

Comment:
We could also get rid of the `$ ` from the command below - https://github.com/nodejs/node/blob/472ffdef73f636cefa0e7015e07499637c34bd43/doc/api/single-executable-applications.md?plain=1#L64